### PR TITLE
kapacitor 0.11.0

### PR DESCRIPTION
**Pull request by gunnaraasen**
### All Submissions:

- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew/pulls) for the same update/change?

### CODE REVIEW

File: Library/Formula/kapacitor.rb [hunk 1]
```diff
@@ -16,26 +16,23 @@ class Kapacitor < Formula
     sha256 "f948ff298c8bdac82b0caefb743a834348d67b38644b2746dea6ae720cca4275" => :mavericks
   end
 
+  option "with-foo", "Compile with foo bindings"
```
Comment (dunn) [inline Library/Formula/kapacitor.rb]: I don't see what this option does?

Comment (DomT4) [inline Library/Formula/kapacitor.rb]: Possibly a documentation copy/paste that was unused; It's the same wording as in our Cookbook.

Comment (gunnaraasen) [inline Library/Formula/kapacitor.rb]: Yup, that's and artifact from fixing up the test.

**Comment by gunnaraasen** [comment]
I've removed the random lines.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/8a07ebcba6f31422583e5e9a86ba7b6a77c8a8ce. Thank you once more @gunnaraasen! :smiley_cat:

**Pull request closed without merging**